plugins/design-system/components/component-example/CodeExampleFooter.js

```diff
@@ -0,0 +1,73 @@
+import classNames from "classnames/dedupe";
+import React, { Component } from "react";
+
+const defaultClasses = {
```

Comment:
I was thinking the same.. but it's a real paradox because we should be creating new components when has something different trying to avoid custom config down the road but at the same time is so minimum that maybe makes sense to have one for both